caliban/util/fs.py

```diff
@@ -122,32 +122,46 @@ class TempCopy(object):
 
   """
 
-  def __init__(self, original_path=None, tmp_name=None):
-    if tmp_name is None:
-      self.tmp_path = ".{}.json".format(str(uuid.uuid1()))
-    else:
-      self.tmp_path = tmp_name
+  class SrcDst(NamedTuple):
```

Comment:
use a dict, vs this reimplementation of dicts

Reply:
ok